### DISCUSSION TITLE
Maybe link About -> Contributors -> Filter lists to wiki licenses page?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2269] [Ideas]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Filter-list-licenses/

And on the wiki page, add a column with link to contributors if possible?

**Comment by gorhill** [comment]
Yeah the link to contributors was me hoping people click the link and realize the amount of work undertaken by volunteers, so if I change the link, it would be good to still navigate to commit graphs.

**Comment by MasterKia** [comment]
@gorhill About -> External Dependencies -> Metropolis font family link is broken

https://github.com/chrismsimpson/Metropolis
=>
https://github.com/dw5/metropolis
